Examples/Algorithms/Seeding/include/ActsExamples/Seeding/SeedingAlgorithm.hpp

```diff
@@ -0,0 +1,96 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Plugins/Digitization/PlanarModuleCluster.hpp"
```

Comment:
I guess we could remove the `PlanarModuleCluster` here and elsewhere since it's not used anymore?